### ISSUE TITLE
2.0.x no longer works with hover/group-hover variants

### ISSUE THREAD

**Issue by bymayo** [adfdev/tailwindcss-bg-alpha#1]
I've got the following in my config:

```
      alphaColors: [
        'primary.900',
        'secondary.900'
      ]
```

Then:

```
  variants: {
    alphaColors: ['hover', 'group-hover']
  },
  plugins: [
    require('tailwindcss-bg-alpha')()
  ]
```

In v1.x of this plugin, the hovers worked. But since v.2.x it doesn't.

I noticed you've added text, bg and border utility classes now which is great news! But this has broken some functionality.

**Comment by adfdev** [comment]
fixed on 2.0.1 release. Get feedback please

**Comment by bymayo** [comment]
@adfdev Thanks!
